src/components/Profile.js

```diff
@@ -57,35 +54,26 @@ class Profile extends Component<Props> {
       && !this.props.trustedByUids.has(this.props.authMemberDoc.id);
   }
 
-  isOwnProfile(uid) {
-    return this.props.authMemberDoc && this.props.authMemberDoc.uid === uid;
+  isOwnProfile() {
+    const { authFirebaseUser, memberDoc } = this.props;
+    return authFirebaseUser && memberDoc && authFirebaseUser.uid === memberDoc.id;
   }
 
-
   render() {
-    const { authFirebaseUser, authIsLoaded, isMyProfile, memberDoc } = this.props;
-    if (!authIsLoaded) {
-      return <Loading />;
-    }
-    if (isMyProfile && !authFirebaseUser) {
-      return (
-        <div>
-          <span>You need to </span>
-          <Redirect to="/login" />
-        </div>
-      );
-    }
-    if (!memberDoc) {
+    const { authFirebaseUser, authIsLoaded, memberDoc } = this.props;
+    if (!authIsLoaded || !memberDoc) {
       return <Loading />;
     }
+    const ownProfile = this.isOwnProfile();
     if (!memberDoc || !memberDoc.get('mid')) {
       if (!this.props.memberId) {
-        // TODO make into it's own <ProfilePending /> component
-        return <FormattedMessage
-          id="invite_pending"
+        return <FormattedMessage  // TODO this does not work
+          id="invite_missing"
           values={{
-            display_name: this.props.authFirebaseUser.displayName,
-            help_email: <a href="mailto:help@raha.io">help@raha.io</a>
+            display_name: authFirebaseUser.displayName,
+            help_email: <a href="mailto:help@raha.io">help@raha.io</a>,
+            login_account: <b>{authFirebaseUser.email}</b>,
+            logout: <Link to='/logout'>logout</Link>
```

Comment:
Same question here - does this link need to be a template var? Is this the only way we can inject html elements into a formatted message?